src/views/InitialSetup.vue

```diff
@@ -109,6 +141,7 @@ export default {
       this.$axios.get("/core/settings/").then((r) => {
         this.allTimezones = Object.freeze(r.data.all_timezones);
         this.timezone = r.data.default_time_zone;
+        this.companyname = r.data.mesh_company_name;
```

Comment:
we don't actually need this line cuz it will be empty on first setup, and you already initialized it to any empty string up in the data section so this is redundant.

Reply:
I had wondered about that one, thanks